src/pages/main_layout.cr

```diff
@@ -110,8 +110,8 @@ abstract class MainLayout
 
     ## What does Lucky look like?
     
-    In Lucky we generate separate action classes with names that describe the route.
-    The resulting file maps routing path definitions to a response block.
+    Lucky can generate **action classes**, from class names that already determine usefull default routing path(s).
```

Comment:
haha. I actually liked it better the other way. Also, it's not that Lucky *can* generate these classes, it *does indeed* generate separate classes. 

After reading this a few times, I'm thinking the original way made a bit more sense, but maybe with a small change.

Maybe something like
```
Lucky generates separate **action classes** that determine routing paths.
The resulting classes map the routing path definition to a response block.
```

Reply:
I wanted to emphasize for newcomers (like me) that one can just tell Lucky to generate an action class of name X:Y:Z, and the name will determine the default routes. I only found this feature out with the old docs after quite a while.

Reply:
What about using "will"  instead of "can" ?

Reply:
oh yeah, for sure. We want to make sure this is as clear and easy to read as possible. Wording is pretty tough since it can be interpreted differently by different people depending on where in the world you come from. I think this is close though.

This reads well for me.
```
Lucky will generate **action classes**, from class names that determine useful default routing path(s).
```

Reply:
For me too.
Also added the "separate" classes in the following explanation.